index.html

```diff
@@ -503,66 +504,64 @@
-information flows. Our goal is not to cover all privacy issues, but rather to provide enough
-background to support the Web community in making informed decisions about privacy and to weave
-privacy into the architecture of the Web. Few architectural principles are absolute, and privacy is
-no exception: privacy can come into tension with other desirable properties of an ethical
-architecture, and when that happens the Web community will have to work together to strike the right
-balance.
+information flows. Our goal is not to cover all possible privacy issues, but rather to provide enough
+background to support the Web community in making informed decisions about privacy and in weaving
+privacy into the architecture of the Web.
+
+Few architectural principles are absolute, and privacy is no exception: privacy can come into tension
+with other desirable properties of an ethical architecture, and when that happens the Web community
+will have to work together to strike the right balance.
 
 Information is power. It can be used to predict and to influence people, as well as to design online
 spaces that control people's behaviour. The collection and [=processing=] of information in greater
 volume, with greater precision and reliability, with increasing interoperability across a growing
 variety of data types, and at intensifying speed is leading to an unprecedented concentration of power
-that threatens private and public liberties.
-
-What's more, automation and the increasing computerisation of all aspects of our lives
-both increase the power of information and decrease the cost of a number of intrusive
+that threatens private and public liberties. What's more, automation and the increasing computerisation
+of all aspects of our lives both increase the power of information and decrease the cost of a number of intrusive
 behaviours that would be more easily kept in check if the perpetrator had to be in the same room as
 the victim.
 
-These asymmetries of information and of automation create commanding asymmetries of power.
+These asymmetries of information and of automation create powerful asymmetries of power.
 
 <dfn data-lt="governance">Data governance</dfn> is the system of [=principles=] that regulate [=information flows=]. When
 [=people=] are involved in [=information flows=], [=data governance=] determines how
 these [=principles=] constrain and distribute the power of information between different [=actors=].
-The <dfn>principles</dfn> describe the way in which different [=actors=] may, must,
+Such <dfn>principles</dfn> describe the ways in which different [=actors=] may, must,
 or must not produce or [=process=] flows of information from, to, or about other [=actors=]
-([[?GKC-Privacy]], [[?IAD]]).
+([[?GKC-Privacy]], [[?IAD]]). Typically, <dfn>actors</dfn> are [=people=] but they can also be collective entities such as
+companies, associations, or governmental bodies, which are known as [=parties=].
```

Comment:
This distinction between "actors" and "parties" doesn't really make sense. I think someone mentioned that we should unify on just one of the terms. But no need to make that happen in this PR that's just moving the sentence between paragraphs.

Reply:
That's what I though while reading it too. I opened #209 to keep track of it.